src/services/localesServices/locales/en.json

```diff
@@ -1,66 +1,42 @@
-	"column \"id\" of relation \"Facilities\" does not exist": "column \"id\" of relation \"Facilities\" does not exist",
 	"you have to be a travel admin to perform this action": "you have to be a travel admin to perform this action",
-	"column \"facilityId\" of relation \"Facilities\" does not exist": "column \"facilityId\" of relation \"Facilities\" does not exist",
-	"null value in column \"id\" violates not-null constraint": "null value in column \"id\" violates not-null constraint",
-	"location is required": "location is required",
-	"destination is required": "destination is required",
-	"reason is required": "reason is required",
-	"accomodation is required": "accomodation is required",
-	"departureDate is required": "departureDate is required",
-	"enter valid date with YYYY-MM-DD format": "enter valid date with YYYY-MM-DD format",
-	"The firstname is required": "The firstname is required",
-	"First name should be only characters": "First name should be only characters",
-	"Last name should be only characters": "Last name should be only characters",
-	"the accomodation doesn't exist or it's not in your destination": "the accomodation doesn't exist or it's not in your destination",
-	"Only USD, EUR or RWF currencies are allowed": "Only USD, EUR or RWF currencies are allowed",
-	"Please choose correct department": "Please choose correct department",
-	"enter vaid date": "enter vaid date",
-	"the accomondation doesn't exist or it's not in your destination": "the accomondation doesn't exist or it's not in your destination",
-	"Request created successfully": "Request created successfully",
-	"request with the same departure date exist": "request with the same departure date exist",
-	"the return date must be greater than departure date": "the return date must be greater than departure date",
-	"User roles updated successfully": "User roles updated successfully",
-	"Invalid value": "Invalid value",
-	"dstination should  only be letter": "dstination should  only be letter",
-	"destination should  only be letter": "destination should  only be letter",
-	"No token provided": "No token provided",
-	"User does not exist": "User does not exist",
-	"User does not exist or they are not a manager": "User does not exist or they are not a manager",
-	"Cannot read property 'role' of null": "Cannot read property 'role' of null",
-	"One or both of the user's ID's do not exist": "One or both of the user's ID's do not exist",
+	"Manager assigned successfully.": "Manager assigned successfully.",
 	"One or both user ID's do not exist": "One or both user ID's do not exist",
-	"User does not exist or they are not a manager or they are both managers": "User does not exist or they are not a manager or they are both managers",
-	"Manager assigned successfully.": "Manager assigned successfully."
+	"User does not exist or they are not a manager or they are both managers": "User does not exist or they are not a manager or they are both managers"
 }
```

Comment:
I believe you removed some messages. We should internationalize every output message.

Reply:
I removed them because they had duplicates...!! If you checked the code, you'd find that there are the same messages..